components/organisation-unit-tree/src/organisation-unit-node/use-org-children.js

```diff
@@ -0,0 +1,56 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import { useMemo, useEffect } from 'react'
+import { sortNodeChildrenAlphabetically } from '../helpers/index.js'
+
+const ORG_DATA_QUERY = {
+    orgUnit: {
+        resource: `organisationUnits`,
+        id: ({ id }) => id,
+        params: {
+            fields: 'children[id,path,displayName]',
+        },
+    },
+}
+
+/**
+ * @param {string[]} ids
+ * @param {Object} options
+ * @param {string} options.displayName
+ * @param {boolean} [options.withChildren]
+ * @returns {Object}
+ */
+export const useOrgChildren = ({
+    node,
+    suppressAlphabeticalSorting,
+    onComplete,
+}) => {
+    const { called, loading, error, data } = useDataQuery(ORG_DATA_QUERY, {
+        variables: { id: node.id },
```

Comment:
This query will not respond to changes in the node. Is it necessary for it to do so? Because if so, a useEffect with the node as a dependency and a refetch that updates the node would address that.

Reply:
The id will always remain the same, so `useEffect` + `refetch` is not necessary here